saleor/saleor-app-slack
# feat: add slack webhook

Hello 👋🏻
This is my first PR to this repository. I wanted to add support for sending slack messages when new order is created in Saleor. This PR resolves two of TODOs from #1 (the first & the second one).

## What are the changes
- feat: send slack message on `order_created` event
- docs: update README & add Slack app tutorial
	- I've updated docs based on [saleor-app-klaviyo](https://github.com/saleor/saleor-app-klaviyo)
- chore: update fields for app
	- Updated based on [saleor-app-klaviyo](https://github.com/saleor/saleor-app-klaviyo)

## How to test this PR
1. Prepare Slack app using this [doc](https://github.com/saleor/saleor-app-slack/blob/feat/add-slack-webhook/docs/setup-slack-app.md)
1. Clone this PR on your local dev machine
2. Run `pnpm i` and `pnpm run dev` + `saleor app tunnel` afterwards. More on dev commands in [Saleor docs](https://docs.saleor.io/docs/3.x/cli#app)
3. Go to your Saleor dashboard - you should be able to see `saleor-app-slack` as third party app
4. Paste your Slack webhook URL in the configuration field and save configuration
5. Create an order e.g using Saleor dashboard
6. See your message on Slack 🎉
